integration_tests/helpers/commonHelpers.ts

```diff
@@ -1,3 +1,12 @@
+import { generateRandomNumber, snsValidEventPayload } from "../payloads/snsEventPayload";
+import { deleteObjectInS3, getS3ItemsList } from "./s3Helper";
+import { publishSNS } from "./snsHelper";
+import { resourcePrefix } from "../helpers/envHelper";
+
+const prefix = resourcePrefix();
+const objectsPrefix = "btm_transactions";
+const details: string[] = [];
```

Comment:
I think it would be cleaner to move this declaration into the `generateTestEventsAndValidateEventExists` function then change `deleteS3Event` to take an array of eventIds as an argument

Reply:
By moving the const details to inside are you suggesting to change the return type of generateTestEventsAndValidateEventExists function and return details ?

Reply:
Yeah I'd do something like that. The way you've done it will work fine but it is prone to bugs if some other function is introduced later that also depends on the details array